css/ace-mantis.css

```diff
@@ -169,6 +169,7 @@
 
 table {
     background-color: #ffffff !important;
+    overflow-wrap: anywhere;
```

Comment:
Unfortunately not supported by all browsers
https://caniuse.com/#feat=mdn-css_properties_overflow-wrap_anywhere

Reply:
Yeah... Unfortunately, CSS compliance remains inconsistent.
The goal is to address the issue for most users and these changes does that while not impacting users who happen to use uncommon browsers or versions.

Reply:
Got aware of it as PhpStorm CSS inspection treats it as `Mismatched property value` error, but this seems to be caused by outdated validation data.